tests/rptest/tests/random_node_operations_test.py

```diff
@@ -326,6 +377,8 @@ def test_node_operations(self, enable_failures, num_to_upgrade,
         # stop producer and consumer and verify results
         regular_producer_consumer.verify()
         compacted_producer_consumer.verify()
+        if with_tiered_storage:
+            self.redpanda.stop_and_scrub_object_storage()
```

Comment:
This should happen automatically at the end of the test via the `cluster` decorator.

Reply:
yes, it does but we stop all the nodes before test finishes to make controller stable hence the explicit check here.

Reply:
That check should run even if the nodes are stopped,

Reply:
```
    def stop_and_scrub_object_storage(self, run_timeout=300):
        # Before stopping, ensure that all tiered storage partitions
        # have uploaded at least a manifest: we do not require that they
        # have uploaded until the head of their log, just that they have
        # some metadata to validate, so that we will not experience
        # e.g. missing topic manifests.
        #
        # This should not need to wait long: even without waiting for
        # manifest upload interval, partitions should upload their initial
        # manifest as soon as they can, and that's all we require.
        # :param run_timeout timeout for the execution of rp-storage-tool.
        # can be set to None for no timeout

        def all_partitions_uploaded_manifest():
            manifest_not_uploaded = []
            for p in self.partitions():
                try:
                    status = self._admin.get_partition_cloud_storage_status(
                        p.topic, p.index, node=p.leader)
```
here the code accesses Redpanda admin APIs, this is why it fails when all nodes are stopped

Reply:
Ah right. I moved that code around in one of my open PRs and I was thinking of that. Makes sense.